### DISCUSSION TITLE
Next steps (technical/modeling volunteer options)

### DISCUSSION THREAD

**Discussion by apiology** [microCOVID/microCOVID#1792] [General]
Please see this post about the current situation for microCOVID:
* https://github.com/microCOVID/microCOVID/discussions/1791

Here are the potential futures I can imagine given the above:

1. No effort: March 11th is the last possible update on the site.  Two weeks from the last update, folks start getting the bright 'Data is stale' warning, but the site defaults to giving bad information based on whatever the last datapoint happened to be in their area.  This is quite possibly a false zero in many areas.  Site stays up until the credit card attached to auto-renew on the domain expires or maybe Netlify drops their 'Covid-19 support' plan and wants money for hosting.
1. Mild frontend effort: Ordered shutdown of site with a note at top, disabling users from further use to avoid the hazard of using that stale data.
1. Moderate frontend effort: We rip out prevalence and locality information entirely from the frontend and update the white-paper accordingly.  The rest of the site stays the same, but now reflects relative risk between different activities and not an estimate of absolute risk.
1. Significant modeling and data engineering effort, mild frontend effort: We move to whatever level of useful data aggregation is left.  I would guess the best case here is limiting data to the US and moving to CDC's information/summaries as the source, but you tell me.  We'd need some effort to figure out how to translate that back into something compatible with our existing model, to determine what if any adjustments would be valuable to add using data from outside sources, and how to best represent to users the current uncertainty level of the data.

What am I missing?  Do you want to claim part of this work?  Join a team working on any of the above, and in what capacity?  I can help coordinate effort, assist with problems you hit, or just review PRs if you want to try something out solo.  I can also help if you think you have relevant skills but don't know where to start—just speak up.

**Comment by jhwgh1968** [comment]
I don't know when I will finish it (probably not by the closure date), but I wanted to mention something I am doing in reaction to this news.

It is another option, perhaps between options 2 and 3: remove the location-based prevalence data and force the user to enter it themselves before the other parts of the form show up.

For a while now, I have overridden the prevalence data for my area, believing it is too low quality. But I am fortunate to live in one of the handful of places which continue to report sewershed data, which I have roughly modeled myself to create a "true prevalence" prediction. That is my input to the site, and seems to mesh well with the site's own conservative modeling.

My predictions are extremely location specific, and my method is neither peer reviewed nor possibly applicable anywhere else, so I see no reason to share it. My point is, I am completely ignoring all prior prevalence data and using my own, and the site is still extremely useful.

I can see both sides of the ethical debate of forcing people to look up and enter their own prevalence data with a "be careful, garbage in means garbage out" warning. I recognize "do your own research" has become a rather unfortunate meme.

But with the site's closure, I will now create my own private fork for continued personal use. I wonder how many other nerdy amateurs will be doing as I do anyway, to varying levels of accuracy and success.

I will let others debate it. I just wanted to mention this as an additional option.

**Reply by chrisvfritz** [reply]
@jhwgh1968 I'm also in an area that continues to report sewershed data and would be eager to see how you're currently using it to model true prevalence, even if your methods aren't peer-reviewed.

**Reply by jhwgh1968** [reply]
@chrisvfritz

It is a really dumb model. The details are too specific to my area, but the logic behind the math is straightforward:

1. Get the current number of viral copies
1. Starting with the the Jan 2022 Omicron spike peak, find the date closest to it (edited) with the same number of copies +/- 5%
1. Assume today's case rate is identical to what it was on that day, back when testing was better

This has a whole host of bad assumptions. It not only assuming the population is stable, but that the virus itself has not changed its ability to bypass countermeasures since Omicron. For the vaccines and lower-quality masks, in particular, this is pretty well shown to be counter-factual.

I simply do not ask it questions where those factors are relevant. But I imagine most people would, which would make my model worse than useless to make public.

**Reply by chrisvfritz** [reply]
Thanks for sharing! 🙏

**Reply by sameerjain0123** [reply]
@apiology @chrisvfritz @jhwgh1968 how accurate do you think microCOVID is currently, with the (fairly subtle) helper text urging users to manually look up weekly new cases per 100k from covidactnow?

If folks think it's reasonably accurate, would it be possible to make that guidance more prominent (@jhwgh1968 's suggestion above - "remove the location-based prevalence data and force the user to enter it themselves before the other parts of the form show up" - would be best, but if we don't have the available skillsets for that, would be helpful to even just put the current text in red and also add a note in red above the November, 17, 2022 update towards the top of the page)?

If folks don't think it's accurate enough to do more good than harm, could we pursue @apiology 's suggestion above ("Mild frontend effort: Ordered shutdown of site with a note at top, disabling users from further use to avoid the hazard of using that stale data.") or if lacking resources for that, add a clear and prominent warning to that effect (perhaps red text in the updates section and under Step 1)?

There are local event venues in SF which are making decisions to loosen safety requirements citing microCOVID data as a source, which makes me think what we currently have in place is misleading. (I don't see a "bright data is stale" warning come up when I run an estimate, contrary to what @apiology mentioned in his "No effort" option #1 in the post at top.)

Unfortunately, I don't have the skills to write any code, but wanted to flag this in case anyone with the relevant skills has bandwidth to pursue some of the lower-effort options.

**Comment by chrisvfritz** [comment]
@apiology Is the [Covid Act Now API](https://apidocs.covidactnow.org/) a possible replacement data source, at least for the US?

**Reply by boxlike-supply** [reply]
I wonder if the Covid Act Now team might be interested in incorporating the microcovid model into their site if it cannot continue independently? https://covidactnow.org/about#contact-us

**Reply by boxlike-supply** [reply]
It looks like the team behind Our World in Data is sourcing alternatives to the Johns Hopkins dataset - discussion started here with a promise of updates: https://github.com/owid/covid-19-data/discussions/2782

**Comment by boxlike-supply** [comment]
I hope there is a path forward for option 3, focused on "relative risk between different activities", which I think could free volunteers from much of the day-to-day maintenance burden but keep the site useful to the public for making objective risk-based decisions (either incorporating user-provided prevalence data, or just creating a generic risk score).

**Reply by Edderic** [reply]
@boxlike-supply I'm the creator of breathesafe.xyz. Breathesafe gives an absolute risk score for venues given duration, activities, *assuming that there is one infector in the room*. It uses CO2 data to estimate ventilation for a location. Then the user can see how adding portable air cleaners and masking contribute to risk. It can also be used for relative risk purposes.

I used to make use of prevalence rates like MicroCOVID did, but at one point started to notice the slowing down of reporting. I thought it will just be a matter of time until data on prevalence becomes less useful & reliable. However, I could bring back that feature. Instead of assuming that the probability of an infector being in a room is 1, we could let the user state a probability, like 1 / 10. Could default to a high ratio, to be on the safer side, but could let the user override the default. I'm also planning to include testing (rapid & PCR), and symptoms, so that if there is some sort of gathering in a venue, people could see how risky the gathering in a comprehensive way (e.g. testing, symptoms, indoor air quality, masking).

Example analysis for a brewery: https://www.breathesafe.xyz/#/analytics/104

Other parts that we're working on: In the analysis page for a venue, write in plain English: "This place does / does not meet ASHRAE, WHO standards for mitigating airborne transmission." So instead of just telling individuals which places are risky or safer, indoor air quality advocates could actually meet with the building manager to enact change (e.g. get air cleaners).

If you're interested, please check us out. Feel free to email info@breathesafe.xyz.

cc: @apiology @jhwgh1968 and others, let me know if you're interested in collaborating.

**Comment by apiology** [comment]
Hey all - a group of contributors has taken up a different option of requiring manual prevalence input.  I'll go ahead and lock this thread.
